modules/nixos/community-builder/users.nix

```
@@ -54,6 +54,11 @@ let
       keys = ./keys/lily;
     };
 
+    networkexception = {
+      trusted = true;
+      keys = ./keys/networkexception;
+    };
```

```suggestion
    networkexception.keys = ./keys/networkexception;
```

We've removed trusted from all users in https://github.com/nix-community/infra/pull/735.